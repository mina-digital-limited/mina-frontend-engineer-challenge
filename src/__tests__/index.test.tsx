import Home from '@/app/page';
import { render, screen } from '@testing-library/react';

describe('GIVEN the Homepage is rendered', () => {
  it('THEN the layout is unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });

  it('THEN the correct heading is displayed', () => {
    render(<Home />);

    const heading = screen.getByText('ğŸ‘‹ Hello World ğŸ‘‹');

    expect(heading).toBeInTheDocument();
  });
});
