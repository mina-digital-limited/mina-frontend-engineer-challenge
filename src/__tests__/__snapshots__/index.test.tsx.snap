// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN the Homepage is rendered THEN the layout is unchanged 1`] = `
<div>
  <main
    class="main"
  >
    <h1>
      ðŸ‘‹ Hello World ðŸ‘‹
    </h1>
  </main>
</div>
`;
